packages/fxa-js-client/package.json

```diff
@@ -6,8 +6,8 @@
   "license": "MPL-2.0",
   "scripts": {
     "start": "grunt",
-    "test": "grunt test",
-    "test-local": "intern-client config=tests/intern auth_server=LOCAL",
+    "test": "mocha tests/lib --reporter dot",
+    "test-local": "AUTH_SERVER_URL=http://127.0.0.1:9000 mocha tests/lib --reporter dot",
```

Comment:
I get some failures running this command.

Reply:
should pass now